### DISCUSSION TITLE
hyprlauncher, a fresh application/file viewer/script launcher

### DISCUSSION THREAD

**Discussion by ghost** [hyprwm/Hyprland#8272] [Show and tell]
https://github.com/nnyyxxxx/hyprlauncher
**Hyprlauncher is a brand new application/file viewer/script launcher made in rust !**
</br>
![image](https://github.com/user-attachments/assets/ffa4db14-b9e4-404c-b4c6-a50a3e3f4204)
![image](https://github.com/user-attachments/assets/1107e6af-c1bf-448c-afd1-23dbabdfca12)

**Comment by abenz1267** [comment]
What's the point? Just fork anyrun?
